test/functional/fixtures/api/es-next/take-screenshot/testcafe-fixtures/take-screenshot.js

```diff
@@ -137,3 +137,11 @@ test('Rewrite a screenshot with warning', async t => {
     await t.takeScreenshot('custom/duplicate.png');
     await t.takeScreenshot('custom/duplicate.png');
 });
+
+test('Take a screenshot with a pathPattern', async t => {
+    await t.takeScreenshot({ pathPattern: '${TEST}/custom' });
+});
+
```

Comment:
I got some unexpected warning message when I tried to run the following test:
```JS
test('Take a screenshot with a pathPattern', async t => {
    await t.takeScreenshot({ pathPattern: '${TEST}/custom' });
    await t.takeScreenshot();
});
```
Warning message is:
```The file at "...\screenshots\Take a screenshot with a pathPattern\custom.png" already exists. It has just been rewritten with a recent screenshot```

Reply:
It could have happened due to the folder 'screenshots' already existing. Maybe, the test failed at some point before, and it did not reach the `return assertionHelper.removeScreenshotDir('screenshots');`. That is why the folder 'screenshots' was not cleared.

Reply:
Actually there is an issue with the current implementation I just found. Please, ignore the above comment. I will let you know once I fix the PR.
Thank you for your comment.